rcp/org.testeditor.rcp4.tcltestrun/src/org/testeditor/rcp4/tcltestrun/TclLauncher.xtend

```diff
@@ -88,4 +100,30 @@ class TclLauncher implements Launcher {
 		return true
 	}
 
+	private def boolean launchMavenBasedTest(IStructuredSelection selection, IProject project, String elementId) {
+		logger.info("Trying to launch maven test execution")
+
+		val job = new Job("Execute test " + elementId) {
+
+			override protected run(IProgressMonitor monitor) {
+				val mvnExec = new MavenExecutor
+
+				val result = mvnExec.executeInNewJvm("integration-test", project.location.toOSString,
+					"test=" + elementId)
+				val testResultFile = project.createOrGetDeepFolder(TclLauncher.MVN_TEST_RESULT_FOLDER).getFile(
+					elementId.elementIdToFileName).location.toFile
+				if (result == 0) {
+					testResultFile.showTestResult
+				} else {
+					logger.error('''Error during maven task "test" of element "«elementId»"''')
```

Comment:
Above the task `integration-test` is called, we should log out the same name here as well, currently `"test"` is logged out in this statement.